Service/MenuService.php

```diff
@@ -150,6 +150,15 @@ public function getProcessMenu(string $path, Process $process = null)
                     'economy_show',
                     ['process' => $process]
                 ),
+                $this->createMenuItem(
```

Comment:
This is outside of the scope of this PR, but building the menu items may be more robust if we match on the current route name rather than on the current path.

Reply:
Good point. I'll make another branch for this.